common-npm-packages/azure-arm-rest-v2/azure-graph.ts

```diff
@@ -12,7 +12,10 @@ export class GraphManagementClient extends azureServiceClient.ServiceClient {
     constructor(credentials: msRestAzure.ApplicationTokenCredentials, baseUri?: any, options?: any) {
         super(credentials, null);
 
-        this.apiVersion = '1.6';
+        const useMSAL = credentials.getUseMSAL();
+        tl.debug(`MSAL - Graph - GraphManagementClient - useMSAL = ${useMSAL}`);
```

Comment:
Is this the debug output, that we can see at the pipeline execution time?

Reply:
It is enabled when `system.debug` parameter is set to `true` in execution. It was a good point and helpful to identify errors.